### DISCUSSION TITLE
v1 feeds.geojson either returns empty response or an error

### DISCUSSION THREAD

**Discussion by newcarrotgames** [transitland/transitland#390] [Q&A] [answered]
Hi, Transitland devs! Hope you had a nice weekend.

I'm using Valhalla as my routing engine. When building the tiles, Valhalla tries to hit the following URL for the transit feeds:

http://transit.land/api/v1/feeds.geojson?per_page=false&api_key=MY_KEY

but this is returning an error:

{
    "message": "[NoMethodError] undefined method `other_ids' for #<Feed:0x00007fb5e0cff6c8>"
}

Can anyone tell me what I'm doing wrong?

EDIT: I should add this request, which always returns an empty response:

http://transit.land/api/v1/feeds?api_key=MY_KEY&bbox=19.64989,-171.56250,71.64872,-63.28125

I've tried all configurations of bbox for my lat/lon values. Your docs say you want southwest corner to northeast corner so that's what I've used as my example; however, the [docs for the feeds API](https://v1.transit.land/documentation/datastore/feeds.html) don't list bbox as a possible parameter in the first place so either this never worked or is just undocumented. I've also tried using a smaller bounding box with the same results.

**Comment by drewda** [comment, marked as answer]
Hi, the Transitland v1 Datastore API is deprecated. We have continued to support most — but not all of its functionality — with a shim in the bridge that powers the v1 API using the v2 infrastructure. We will be shutting this down in the future. All users should switch to using the v2 APIs as soon as possible.

We're aware that Valhalla still only makes use of Transitland's v1 APIs but that is outside of the scope of the Transitland platform. We're sorry this is probably not the answer you want to hear, but we want to be straightforward — Transitland v2 APIs are the ones for which we provide support and are making feature and performance improvements.

**Reply by newcarrotgames** [reply]
Should I just close https://github.com/transitland/transitland/discussions/391?
